path_config.go

```diff
@@ -11,44 +11,52 @@ func pathConfig(b *azureAuthBackend) *framework.Path {
 	return &framework.Path{
 		Pattern: "config",
 		Fields: map[string]*framework.FieldSchema{
-			"tenant_id": &framework.FieldSchema{
+			"tenant_id": {
 				Type: framework.TypeString,
 				Description: `The tenant id for the Azure Active Directory.  This is sometimes
-				referred to as Directory ID in AD.  This value can also be provided with the 
+				referred to as Directory ID in AD.  This value can also be provided with the
 				AZURE_TENANT_ID environment variable.`,
```

Comment:
The extra indents that were in place before this PR are also unnecessary. This was also taken into consideration when the help description was being rendered (since we're using string literals via backticks). Can we have these removed entirely for all multi-line descriptions?

```suggestion
referred to as Directory ID in AD.  This value can also be provided with the
AZURE_TENANT_ID environment variable.`,
```